packages/core/src/services/chains/run.ts

```diff
@@ -88,29 +116,31 @@ export async function runChain({
     stream,
     response,
     resolvedContent: chain.rawText,
-    documentLogUuid,
```

Comment:
veeery weird that we return errorableUuid, honestly I would return both props even if they are the same